test/unit/calculators/check_travel_during_coronavirus_calculator_test.rb

```diff
@@ -120,5 +120,52 @@ class CheckTravelDuringCoronavirusCalculatorTest < ActiveSupport::TestCase
       @calculator.countries = %w[spain]
       assert @calculator.single_journey?
     end
+
+    context "summary_text_fields" do
+      should "return an empty array if only travelling to ireland, even with other fields" do
+        @calculator.countries = %w[ireland]
+        @calculator.travelling_with_young_people = "yes"
+
+        assert_equal [], @calculator.summary_text_fields
+      end
+
+      should "not include young person if not travelling_with_young_people?" do
+        @calculator.travelling_with_young_people = "no"
+
+        assert_not @calculator.summary_text_fields.include?("young_person")
+      end
+
+      should "include responses to the travelling_with_children question" do
+        @calculator.travelling_with_children = "zero_to_four,five_to_seventeen"
+
+        %w[zero_to_four five_to_seventeen].each do |age|
+          assert @calculator.summary_text_fields.include?(age)
+        end
+      end
+
+      should "include the correct vaccination status for unvaccinted users" do
+        @calculator.vaccination_status = "9ddc7655bfd0d477"
+
+        assert @calculator.summary_text_fields.include?("not_vaxed")
+      end
+
+      should "include the correct vaccination status for vaccinated users" do
+        @calculator.vaccination_status = "529202127233d442"
+
+        assert @calculator.summary_text_fields.include?("fully_vaxed")
+      end
+
+      should "include young person if travelling_with_young_people?" do
+        @calculator.travelling_with_young_people = "yes"
+
+        assert @calculator.summary_text_fields.include?("young_person")
+      end
```

Comment:
I think this "young person" test should either be moved next to other test for young people, or replace it. For a moment I thought I was looking at a duplicate test!

Reply:
sorted! Sorry!